### PR TITLE
[INLONG-2600][Manager] Rename the third party cluster class name and table name

### LINKED ISSUE #2600: [Improve][Manager] Rename the third party cluster class name and table name
### Description

# Description
* Rename database table `cluster_info` to `third_party_cluster`.
* Rename some java classes, SQL files, and mapper files of operating third-party cluster. 
* Adjusting API format of managing the third-party cluster.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by herywang**
Fixes #2600

### Motivation

Rename third-party cluster management class name, database table name, and API name.

**examples:**

**table name:** `cluster_info -> third_party_cluster`
**class name:** `ClusterInfo -> ThirdPartyCluster`
**API :** `/common/list -> /thirdparty/list`

### Modifications
None

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
the commit message is not specific, please highlight the changed objects.

**Comment by herywang** [comment]
>

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/ThirdPartyClusterInfoEntity.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 import java.util.Date;
 
 @Data
-public class ClusterInfoEntity implements Serializable {
+public class ThirdPartyClusterInfoEntity implements Serializable {
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/ThirdPartyClusterInfoEntity.java:26]: Please change it to `ThirdPartyClusterEntity`.

File: inlong-manager/manager-dao/src/main/resources/generatorConfig.xml [hunk 2]
```diff
@@ -114,7 +114,7 @@
                 enableCountByExample="false" enableDeleteByExample="false"
                 enableSelectByExample="false" enableUpdateByExample="false"/>
 
-        <table tableName="cluster_info" domainObjectName="ClusterInfoEntity"
+        <table tableName="third_party_cluster" domainObjectName="ClusterInfoEntity"
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/resources/generatorConfig.xml:117]: The `domainObjectName` should change at the same time.

Comment (herywang) [reply inlong-manager/manager-dao/src/main/resources/generatorConfig.xml:117]: I got it.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/ThirdPartyClusterMapper.java [hunk 3]
```diff
@@ -19,28 +19,28 @@
...
 import org.apache.inlong.manager.common.pojo.cluster.ClusterRequest;
-import org.apache.inlong.manager.dao.entity.ClusterInfoEntity;
+import org.apache.inlong.manager.dao.entity.ThirdPartyClusterEntity;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
 
 @Repository
-public interface ClusterInfoMapper {
+public interface ThirdPartyClusterMapper {
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/ThirdPartyClusterMapper.java:28]: Please change it to `ThirdPartyClusterEntityMapper`.

**Review by healchow — APPROVED** [review]
LGTM.

**Pull request merged**